23/README.md

```diff
@@ -0,0 +1,283 @@
+0.680    2.898000
+0.950    5.402000
+0.997    7.686003
+Name: d_commit, dtype: float64
+```
+BigchainDB was able to process 1,000,000 transactions in 26 minutes without any failure.
+
+In terms of responsiveness, the system performed consistently during the whole test:
+- 99.7% of transactions have been **accepted** within 2.033 seconds, 95% within 0.113 seconds, and 68% within 0.018 seconds.
+- 99.7% of transactions have been **finalized** within 7.686 seconds, 95% within 5.402 seconds, and 68% within 2.898 seconds.
+
+In terms of speed (transactions per second), the system:
+- **Accepted** an average of _636 transactions per second_, with a median value of _615 transactions per second_.
+- **Finalized** an average of _636 transactions per second_, with a median value of _636 transactions per second_.
+
+
+
+### Experiment 3: finalize 16,000 transactions (experimental parallelization)
+![Experiment 3][self:plot-experiment-3]
+
+Together with the plots, this output was captured:
+
+```
+0.680    0.014
+0.950    0.074
+0.997    0.373
+Name: d_accept, dtype: float64
+0.680    3.081000
+0.950    4.099000
+0.997    4.358003
+Name: d_commit, dtype: float64
+```
+
+BigchainDB was able to process 16,000 transactions in 18 seconds without any failure.
+
+In terms of responsiveness, the system performed consistently during the whole test:
+- 99.7% of transactions have been **accepted** within 0.373 seconds, 95% within 0.074 seconds, and 68% within 0.014 seconds.
+- 99.7% of transactions have been **finalized** within 4.358 seconds, 95% within 4.099 seconds, and 68% within 3.081 seconds.
+
+In terms of speed (transactions per second), the system:
+- **Accepted** an average of _1092 transactions per second_, with a median value of _1072 transactions per second_.
+- **Finalized** an average of _889 transactions per second_, with a median value of _1102 transactions per second_.
+
+
+## Discussion
+<!--DISCUSSION. The discussion section will explain the authors interpret their data and how they connect it to other work.
+
+Authors often use the discussion to describe what their work suggests and how it relates to other studies. In this section, authors can anticipate and address any possible objections to their work. The discussion section is also a place where authors can suggest areas of improvement for future research.-->
+
+
+The aims of the study are twofold, on one side we want to have an understanding on how a BigchainDB network performs under stress; on the other side we want to define a methodology we can use for a) future versions of BigchainDB, b) running other tests under different conditions.
+
+About the performance of the system, we can conclude that BigchainDB performs consistently and correctly under constant stress, and it's able to finalize one million transactions within one hour and always be responsive. Enabling parallel validation allows BigchainDB to finalize the same number of transactions less than half an hour.
```

Comment:
`finalize the same number of transactions less than half an hour.`

should be

`finalize the same number of transactions in less than half an hour.`